<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>密保问题</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_pwd">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>123456</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>性  别</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>身份证号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cb_sex">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="currentText">
        <string>男</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="le_phone">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>18246514717</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_checkPwd">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>123456</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="le_email">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>1036982019@qq.com</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_username">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>ghost</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>确认密码</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>手机号</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>邮  箱</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>年  龄</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>密保答案</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="le_secret">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>1 + 2 = ?</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="le_age">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>密  码</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="le_answer">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="le_id">
       <property name="styleSheet">
        <string notr="true">border-radius:10px;
border:1px solid;</string>
       </property>
       <property name="text">
        <string>230126000000001111</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btn_cancel">
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:2px solid;</string>
      </property>
      <property name="text">
       <string>返  回</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_register">
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:2px solid;</string>
      </property>
      <property name="text">
       <string>注  册</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lb_waring">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
